src/interface/qt/AccountActivity.cpp

```diff
@@ -238,6 +241,58 @@ auto AccountActivityQt::validateAmount(const QString& amount) const noexcept
     return imp_->parent_.ValidateAmount(amount.toStdString()).c_str();
 }
 
+QString AccountActivityQt::dumpData(const BlockchainAccountStatusQt* accountStatus) const noexcept
+{
+    QDateTime dateTime = dateTime.currentDateTime();
+    QString currentDateTime = dateTime.toString("MM-dd-yy hh_mm_ss");
+    std::string fileName {"transaction-dump_" + currentDateTime.toStdString() +
+                           ".csv"};
+    std::ofstream outfile(fileName);
+    std::stringstream stream;
+    auto transactionCount = rowCount();
+    // Is there a better way to get currencySign?
+    auto currencySign =
+        display::GetDefinition(imp_->parent_.Unit()).Format(Amount(0));
+
+    auto ix = currencySign.find(" ");
+    if (ix < currencySign.size()) {
+        currencySign = currencySign.substr(ix + 1, currencySign.size());
+    }
+
+    stream <<currencySign.c_str() << " Transactions: " << transactionCount <<
+    " Time: " << currentDateTime.toStdString().c_str() << "\n";
```

Comment:
in `Test_DisplayScales.cpp` you can see reference test `TEST(DisplayScale, usd_scales)`on how to use scales when they are defined as in 
`src/core/display/Definition.cpp` in `auto GetDefinition(UnitType in) noexcept -> const Definition&`

i believe you could do the same here ;)